src/deb/init.d/jmxtrans

```diff
@@ -1,30 +1,30 @@
 #!/bin/bash
 #
-# /etc/init.d/@PACKAGE -- startup script for @PACKAGE
+# /etc/init.d/jmxtrans -- startup script for jmxtrans
```

Comment:
Removing `@PACKAGE` seems like a good idea to me. This level of indirection was not really required as there is no reason that this name will change (and changing it will break all kind of things). +1